src/SFA.DAS.AANHub.Application/Common/Validators/ApprenticeId/IApprenticeId.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.AANHub.Application.Common.Validators.ApprenticeId
+{
+    public interface IApprenticeId
```

Comment:
I think the functionality you've implemented with this interface can be implemented without an interface. Otherwise we're going to have one interface per Id per User.

Plus, interfaces act as more of a "contract" for the model to implement the requested components for multiple different models, and this is probably the only place it will be used.

Reply:
Reverted to previous state